### PR TITLE
Add ES suports on Studio

### PR DESCRIPTION
**Pull request by dhaneshnm**
- Save ES view on a works space.
- Edit ES queries in dashboards.
- View results in a table view

### CODE REVIEW

**Comment by kenjinp** [comment]
Users can now visualize ES queries saved via editing the workspace. 
This PR doesn't change the studio workspace creation and editor forms, which still needs work (eg. preventing changing view types on existing workspaces and letting them choose one on creation)

The ES query is creating by combining the dashboard query with another query generated for the purposes of pagination / sorting.

**Review by kenjinp — DISMISSED** [review]

File: src/subapps/studioLegacy/containers/DashboardResults/DashboardElasticSearchQueryContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+}> = ({ view, dataQuery, goToStudioResource }) => {
+  const [searchResponse, { searchProps, setSearchProps }] = useSearchQuery(
+    view._self
+  );
+
+  const handleClickItem = (resource: Resource) => {
+    goToStudioResource(resource._self);
+  };
+
+  const handlePagniationChange = (page: number, pageSize?: number) => {
```
Comment (Stafeeva) [inline src/subapps/studioLegacy/containers/DashboardResults/DashboardElasticSearchQueryContainer.tsx:19]:
```suggestion
  const handlePaginationChange = (page: number, pageSize?: number) => {
```

File: src/subapps/studioLegacy/containers/DashboardResults/DashboardElasticSearchQueryContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      searchResponse={searchResponse}
+      onClickItem={handleClickItem}
+      pagination={
+        shouldShowPagination
+          ? {
+              total,
+              current,
+              pageSize: size,
+              showSizeChanger: false,
+              onChange: handlePagniationChange,
```
Comment (Stafeeva) [inline src/subapps/studioLegacy/containers/DashboardResults/DashboardElasticSearchQueryContainer.tsx:49]:
```suggestion
              onChange: handlePaginationChange,
```

File: src/subapps/studioLegacy/containers/DashboardResults/DashboardSparqlQueryContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+  return (
+    <Spin spinning={queryResult.loading}>
+      {queryResult.error && (
+        <Alert
+          message="Error loading dashboard"
+          description={`Something went wrong. ${(queryResult.error as NexusSparqlError)
+            .reason || (queryResult.error as Error).message}`}
+          type="error"
+        />
+      )}
```
Comment (Stafeeva) [inline src/subapps/studioLegacy/containers/DashboardResults/DashboardSparqlQueryContainer.tsx:137]: It looks like an unnecessary duplicate of the Alert above?

**Review by Stafeeva — APPROVED** [review]
🥬

**Review by kenjinp — APPROVED** [review]

**Pull request merged**